src_c/font.c

```diff
@@ -664,11 +696,10 @@ static PyMethodDef font_methods[] = {
      DOC_FONTGETSTRIKETHROUGH},
     {"set_strikethrough", font_set_strikethrough, METH_O,
      DOC_FONTSETSTRIKETHROUGH},
-
```

Comment:
really minor nitpick, but not sure why these whitespaces got removed

Reply:
Because they shouldn't have been there in the first place? They seem fully extraneous to me.

Reply:
I think they made sense, but it does not matter that much